backend/apid/routers/authentication.go

```diff
@@ -33,69 +37,44 @@ func (a *AuthenticationRouter) login(w http.ResponseWriter, r *http.Request) {
 	// Check for credentials provided in the Authorization header
 	username, password, ok := r.BasicAuth()
 	if !ok {
-		http.Error(w, "Request unauthorized", http.StatusUnauthorized)
+		http.Error(w, http.StatusText(http.StatusUnauthorized), http.StatusUnauthorized)
 		return
 	}
 
 	// Authenticate against the provider
-	user, err := a.store.AuthenticateUser(r.Context(), username, password)
+	claims, err := a.authenticator.Authenticate(r.Context(), username, password)
 	if err != nil {
-		logger.WithField(
-			"user", username,
-		).WithError(err).Error("invalid username and/or password")
-		http.Error(w, "Request unauthorized", http.StatusUnauthorized)
+		logger.WithField("user", username).Error("invalid username and/or password")
```

Comment:
Does the error in this case contain sensitive information?

Reply:
Was just curious why we weren't logging the specific error. If you don't think its necessary, that's fine too!

Reply:
Oh my bad, I misread the code; we _should_ be logging the error, my bad!